alerts/cluster-monitoring-operator/PrometheusRemoteStorageFailures.md

```diff
@@ -0,0 +1,47 @@
+# PrometheusRemoteStorageFailures
+
+## Meaning
+
+The `PrometheusRemoteStorageFailures` alert triggers when there has been
+a constant increase in failures sending samples to the remote storage for more
+than 15 minutes, in either the platform Prometheus or a user-defined Prometheus
+instance.
+
+## Impact
+
+Prometheus samples in remote storage might be missing or inaccurate.
+
+## Diagnosis
+
+1. Determine if the alert has been triggered for the instance of Prometheus
+   instance used for default cluster monitoring or for the instance that
+   monitors user-defined projects by checking the alert message's `namespace`
+   label. The namespace value will indicate the Prometheus instance.
+
+   E.g. `openshift-monitoring` for default platform monitoring and
+   `openshift-user-workload-monitoring` for user-defined monitoring.
+
+2. Review the logs for the affected Prometheus instance:
+
+   ```console
+   $ NAMESPACE='<value of namespace label from alert>'
```

Comment:
The alert should also carry the pod name which identifies exactly which pod to look at.

Reply:
I'm already pointing to the`app.kubernetes.io/component=prometheus` label, would you specify anything further?